addons/html_editor/static/src/fields/html_field.js

```diff
@@ -286,7 +286,7 @@ export class HtmlField extends Component {
 export const htmlField = {
     component: HtmlField,
     displayName: _t("Html"),
-    supportedTypes: ["html"],
+    supportedTypes: ["html", "text", "char"],
```

Comment:
The html field should support only the field type "html".
You can modify the field to the correct type, or remove the widget from the view.

Reply:
Please allow text as before, as we have some use case to support in mail.